From 260b02bdb569c36da14f30229a2fa7dc208dc41b Mon Sep 17 00:00:00 2001
From: Martchus <martchus@gmx.net>
Date: Sun, 18 Sep 2016 17:59:27 +0200
Subject: [PATCH 16/32] Enable rpath for build tools

- Required because various tools depend on libQt5Bootstrap.so which resides
  in folder /usr/${_arch}/lib
- Can't use regular lib dir because it would conflict with the native package

Change-Id: I9808062a66406cdec7446a75150b40b2f30038c7
---
 mkspecs/features/qt_app.prf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mkspecs/features/qt_app.prf b/mkspecs/features/qt_app.prf
index 8354f30eea..30e8cf18c5 100644
--- a/mkspecs/features/qt_app.prf
+++ b/mkspecs/features/qt_app.prf
@@ -21,6 +21,7 @@ isEmpty(QMAKE_TARGET_DESCRIPTION): \
 isEmpty(QMAKE_INFO_PLIST): CONFIG -= app_bundle
 
 host_build: QT -= gui   # no host tool will ever use gui
+host_build: QMAKE_RPATHDIR += $$[QT_INSTALL_LIBS/dev] # force rpath
 host_build:force_bootstrap {
     !build_pass:qtConfig(release_tools): CONFIG += release
     contains(QT, core(-private)?|xml) {
@@ -31,7 +32,6 @@ host_build:force_bootstrap {
 } else {
     !build_pass:qtConfig(debug_and_release): CONFIG += release
     target.path = $$[QT_INSTALL_BINS]
-    CONFIG += relative_qt_rpath  # Qt's tools and apps should be relocatable
 }
 INSTALLS += target
 
-- 
2.39.0

